docs/live-query.rst

```diff
@@ -0,0 +1,176 @@
+Live Query
+==========
+
+With Live Query, you can ask questions of endpoints and quickly identify areas for improving security and IT hygiene.
+
+You can use recommended queries created by Carbon Black security experts or craft your own SQL queries. Live Query is
+powered by https://osquery.io, an open source project that uses an SQLite interface. This guide will get you started
+using Live Query via the Python SDK.
+
+More information about the Audit and Remediation product which uses Live Query is available in the
+`Carbon Black Cloud user guide
+<https://docs.vmware.com/en/VMware-Carbon-Black-Cloud/services/carbon-black-cloud-user-guide/GUID-129D4F84-1BF0-49F3-BF95-83002FD63217.html/>`_
+
+More information about Live Query APIs is available on the `Developer Network <https://developer.carbonblack.com/reference/carbon-black-cloud/cb-liveops/>`_.
+
+Overview
+--------
+This guide shows how to find specific files on a system. This is the same scenario as the Quick Start Guide for the
+APIs on the `Developer Network <https://developer.carbonblack.com/reference/carbon-black-cloud/cb-liveops/latest/livequery-api/>`_
```

Comment:
You can point the link directly to the Quick Start section: https://developer.carbonblack.com/reference/carbon-black-cloud/cb-liveops/latest/livequery-api/#quick-start

And a note: I have added two more quick start guides for Live Query that will be available with the DIfferential Analysis API. 
1. Find Installed Firefox add-ons
2. Automatically Recurring Query

Reply:
thanks.  I'll make a note to update this guide when Diff Analysis is added.